plugins/by-name/kitty-scrollback/default.nix

```
@@ -0,0 +1,14 @@
+{ lib, ... }:
+lib.nixvim.plugins.mkNeovimPlugin {
+  name = "kitty-scrollback";
+  package = "kitty-scrollback-nvim";
+
+  maintainers = [ lib.maintainers.nim65s ];
+
+  settingsExample = {
+    # create your config
+    myconfig.kitty_get_text.ansi = false;
+    # or extend a builtin one
+    ksb_builtin_last_cmd_output.ansi = false;
+  };
```

If these comments are intended to be shown in documentation, then you need:

```suggestion
  settingsExample = lib.literalExpression ''
    {
      # create your config
      myconfig.kitty_get_text.ansi = false;
      # or extend a builtin one
      ksb_builtin_last_cmd_output.ansi = false;
    }
  '';
```